# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gcombust/gcc4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur gcombust-0.1.55.orig/src/optimize_usage.c gcombust-0.1.55/src/optimize_usage.c
--- gcombust-0.1.55.orig/src/optimize_usage.c	2003-06-30 16:07:27.000000000 +0300
+++ gcombust-0.1.55/src/optimize_usage.c	2005-11-01 20:47:11.000000000 +0200
@@ -31,7 +31,7 @@
 #include "locks.h"
 #include "globals.h"
 
-static GtkWidget *opt_try_harder;
+GtkWidget *opt_try_harder;
 
 static GtkWidget *cdsize_option_menu, *cdsize_selected_label, *cdsector_selected_label;
 static GtkWidget *used_size_label, *used_sectors_label, *wasted_size_label, *wasted_sectors_label;
diff -Nur gcombust-0.1.55.orig/src/rip_audio.c gcombust-0.1.55/src/rip_audio.c
--- gcombust-0.1.55.orig/src/rip_audio.c	2002-02-16 22:00:52.000000000 +0200
+++ gcombust-0.1.55/src/rip_audio.c	2005-11-01 20:47:40.000000000 +0200
@@ -41,7 +41,7 @@
 char *cdda2wav_path = NULL, *cdda_device_path;
 char *def_rip_path = NULL;
 char *cdparanoia_path = NULL, *cdpara_device_path;
-static GtkWidget *rip_path_entry;
+GtkWidget *rip_path_entry;
 
 static GtkWidget *rip_win = NULL, *p_win;
 static GtkWidget *autoscroll_cdda2wav_text, *cdda2wav_text, *cdda2wav_text_scrollbar;
